1-org/modules/centralized-logging/outputs.tf

```diff
@@ -29,6 +34,17 @@ output "logbucket_destination_name" {
   value       = try(module.destination_logbucket[0].resource_name, "")
 }
 
+output "log_sink_names" {
+  value = [
+    for key, options in local.destinations_options : "${coalesce(options.logging_sink_name, local.logging_sink_name_map[key])}-billing-${random_string.suffix.result}"
+  ]
```

Comment:
this should be a map like
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/outputs.tf#L96C1-L99C2